packages/design-system/src/components/contextMenu/menu.js

```diff
@@ -113,33 +126,64 @@ const Menu = ({
         return;
       }
 
-      const focusableChildren = getFocusableChildren(menuRef.current);
+      const focusableChildren = getFocusableChildren(
+        menuRef.current,
+        isSubMenu
+      );
 
-      const prevIndex = focusableChildren.findIndex(
+      let prevIndex = focusableChildren.findIndex(
         (element) => element.id === focusedId
       );
 
       if (prevIndex === -1 && focusableChildren.length) {
         setFocusedId(focusableChildren[0].id);
-        return;
+        prevIndex = 0;
       }
 
-      const isAscending = [KEYS.UP, KEYS.LEFT].includes(key);
-      let newIndex = prevIndex + (isAscending ? -1 : 1);
+      const prevItemHasSubMenuOpen =
```

Comment:
Not sure where exactly to add this comment, so adding it here because why not.

When you focus the parent item with a submenu and press enter to open the submenu, the first item in the submenu should receive focus, the focus should not remain on the parent item as it does now.

Reply:
@samwhale Currently, when the submenu is opened, the first item is receiving focus, however, it doesn't have visible focus. Any thoughts on how to make sure that it also has visible focus? Tried [using `setFocusedId`](https://github.com/google/web-stories-wp/pull/10199/files#diff-4b4f2ec27b2ca00e1e22c7773f5269fc5ff36c65681344d1af0fe60bbf3a56d8R103) but without luck:
```
      subMenuItems[0].focus();
      setFocusedId(subMenuItems[0].id);
```